samples/Samples.SqlServer/Properties/launchSettings.json

```diff
@@ -11,6 +11,7 @@
         "CORECLR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
         "CORECLR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\Datadog.Trace.ClrProfiler.Native.dll",
 
+        "DD_DOTNET_TRACER_HOME": "$(ProjectDir)$(OutputPath)profiler-lib",
```

Comment:
Don't we need to add this in every sample app?

Reply:
Yes we'll need this for all apps so we can test in VS. This PR only converts a handful of the sample apps to the new mechanism so that we are still testing the "NuGet" scenario where the assemblies are loaded from the bin directory and do not invoke our AssemblyResolve event. Should I convert all sample apps in this PR to use this approach?

Reply:
I just worried that when this PR is merged, running some apps from VS will load the profiler and others will not. Then we spend time troubleshooting the profiler until we realize it's a missing env var.

Reply:
Ok, I can add the DD_DOTNET_TRACER_HOME env var to all of the `launchSettings.json` files so we don't run into that issue.

Reply:
Fixed in be14e00afdacd21136a502cc8bf57e8e1530db60. Also, this edits the MSI to add the `DD_DOTNET_TRACER_HOME` environment variable which I forgot to do before.